start-all.sh

```diff
@@ -266,6 +278,10 @@ fi
 # Can't use localhost here, because the monitoring may be running remotely.
 # Also note that the port to which we need to connect is 9090, regardless of which port we bind to at localhost.
 DB_ADDRESS="$(docker inspect --format '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' $PROMETHEUS_NAME):9090"
+if [ ! -z "$is_podman" ] && [ "$DB_ADDRESS" = ":9090" ]; then
+    HOST_IP=`hostname -I | awk '{print $1}'`
+    DB_ADDRESS="$HOST_IP:9090"
+fi
```

Comment:
What does this change fix? It needs to be explained either in the code as a comment, or in  a commit message.
Also, what does "hostname -I | awk '{print $1'}" even trying to do? The hostname manual page clearly says: "Do not  make  any  assumptions about the order of the output" - so what's the meaning of taking the first one?